klc-cert-manager/Makefile

```diff
@@ -0,0 +1,213 @@
+
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
+# RELEASE_REGISTRY is the container registry to push into.
+RELEASE_REGISTRY?=ghcr.io/keptn
+RELEASE_TIME=$(shell date +%Y%m%d%s)
+BUILD_TIME=$(shell date -u "+%F_%T")
+RELEASE_VERSION?=$(RELEASE_TIME)-v0.24.3#$(shell git describe --tags --match "v*")
+TAG?=latest
+RELEASE_IMAGE:=keptn-cert-operator:$(TAG)
```

Comment:
why do we have an `IMG` and `RELEASE_IMAGE` var?

Reply:
copypasta